app/enrichment/get-frn.js

```diff
@@ -1,8 +1,10 @@
 const db = require('../data')
 
 const getFrn = async (sbi, transaction) => {
-  const frn = await db.frn.findOne({ where: { sbi } }, { transaction })
-  return Number(frn?.frn ?? 0)
+  if (sbi) {
+    const frn = await db.frn.findOne({ where: { sbi } }, { transaction })
+    return Number(frn?.frn ?? 0)
```

Comment:
This threw me a bit when updating as wasn't really sure why we were defaulting to `0` if there was no FRN rather than `undefined` like all other fields.

The end goal from this is to add an FRN if a mapping is found.  If one is not found then reject the request.   Because both `0` and `undefined` fail the validation rules it technically doesn't matter, but I'd rather it was consistent.  

Maybe it was just a way to avoid `Number` returning `NaN`?  If so pretty simple to tweak to use undefined and be consistent.  Let me go back through the commits and see if I can figure it out....

Reply:
Commits in this repo are a bit misleading as they only really start from when we split the service up into multiple services rather than the single monolith we started with.

But anyway, the value 0 isn't a dependency on anything else so I've rewritten it to return `undefined` if no match found.